### PR TITLE
LPS-131988 | master SAML test fixes using UI login

### PR DESCRIPTION
**Pull request by gaborlovasdotcom**
Hi @JoyceWang08211 !
Could you please review my UI login modifications for SAML.testcase and SAMLUpgrade.testcase?

https://issues.liferay.com/browse/LPS-131988

Recent results in [Testray](https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1532524276&testrayRunId=1532524909)

cc: @xbrianlee

### CODE REVIEW

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:saml

**Comment by gaborlovasdotcom** [comment]
ci:test:saml

File: portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro [hunk 1]
```diff
@@ -182,23 +185,23 @@ definition {
...
 	macro executeSPInitiatedSSO {
 		if ("${autoLogin}" == "true") {
-			Navigator.openSpecificURL(url = "${specificURL}");
+			Navigator.openSpecificURL(url = "${spURL}");
 
 			AssertElementPresent(locator1 = "UserBar#USER_AVATAR_IMAGE");
 		}
 		else {
-			User.loginPG(
+			User.firstLoginUI(
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro:193]: Don't forget to remove useless parameters here.

Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro:193]: Don't forget to remove useless parameters here. cc @gaborlovasdotcom

Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro:193]: Cleaned up

**Comment by JoyceWang08211** [comment]
ci:test:saml

**Comment by JoyceWang08211** [comment]
ci:test:sf

**Comment by JoyceWang08211** [comment]
New testray result: https://testray.liferay.com/home/-/testray/case_results/index?p_p_state=normal&orderByCol=status_sortable&orderByType=asc&testrayBuildId=1532860835.

Need to keep tracking LocalFile.SAML#ConfigureSAMLToImportUsersFromLDAP.

Resolve SF issue and forward to BChan under:  https://github.com/JoyceWang08211/liferay-portal/pull/283

**Pull request closed without merging**